workshop/session_3/11_biopython_proteins.ipynb

```diff
@@ -0,0 +1,957 @@
+      " 'T': 83,\n",
+      " 'V': 90,\n",
+      " 'W': 23,\n",
+      " 'Y': 40}\n"
+     ]
+    }
+   ],
+   "source": [
+    "count_of_aas = analysis.count_amino_acids() \n",
+    "print(\"Count of particular aminoacids\")\n",
+    "print(count_of_aas)\n",
+    "print(\"Using PrettyPrint for more user-friendly representation.\")\n",
+    "import pprint\n",
+    "pprint.pprint(count_of_aas)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "editable": false
+   },
+   "source": [
+    "Let's take a look at some protein properties available.\n",
+    "\n",
+    "`\"{:.2f}\"` is used to print only a `float` number to the first two decimal places."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 163,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Molecular weigth    : 168139.53\n",
+      "Charge at a given pH: 38.97\n",
+      "Isoelectric point   : 8.91\n",
+      "Fractions of AA associated with secondary structures:\n",
+      "  Helix: 0.36\n",
+      "  Turn: 0.21\n",
+      "  Sheet: 0.27\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Molecular weigth    :\", \"{:.2f}\".format(analysis.molecular_weight()))\n",
+    "print(\"Charge at a given pH:\", \"{:.2f}\".format(analysis.charge_at_pH(5.8)))\n",
+    "print(\"Isoelectric point   :\", \"{:.2f}\".format(analysis.isoelectric_point()))\n",
+    "in_helix, in_turn, in_sheet = analysis.secondary_structure_fraction()\n",
+    "print(\n",
+    "    \"Fractions of AA associated with secondary structures:\\n\"\\\n",
+    "    \"  Helix: {:.2f}\\n\"\\\n",
+    "    \"  Turn: {:.2f}\\n\"\\\n",
+    "    \"  Sheet: {:.2f}\\n\".format(in_helix, in_turn, in_sheet)\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "editable": false
+   },
+   "source": [
+    "We can also use the helper functions provided to create new statistics. \n",
+    "\n",
+    "For example, let's calculate BCAA (branch-chain amino acid) content of the protein."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 164,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "BCAA content: 0.2648648648648649\n"
+     ]
+    }
+   ],
+   "source": [
+    "total_number_of_LIV_aas = sum(count_of_aas[aa] for aa in [\"L\", \"I\", \"V\"])\n",
+    "print(\"BCAA content:\", total_number_of_LIV_aas / len(cftr_aa))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "editable": false
+   },
+   "source": [
+    "## Property prediction\n",
+    "In this section, we will analyse the hydrophobicity of the protein. \n",
+    "\n",
+    "[The Kyte-Doolittle scale](https://doi.org/10.1016/0022-2836(82)90515-0) is useful for predicting the hydropathic character of the molecule and is based on the experimentally - derived aminoacid properties as defined below."
```

Comment:
Link is not formatted correctly? (Doesn't work in my notebook), The brackets in the link probably need to be escaped?

Reply:
Might also be worth mentioning what the scale means (they probably will not read the paper)

Reply:
I was able to open the link in Binder and on my local (yes, the non-escaping is a bit sketchy) .
Added the description.